release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -495,6 +495,13 @@ The egress router CNI plug-in is generally available. The Cluster Network Operat
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/using-an-egress-router-ovn.adoc[Considerations for the use of an egress router pod].
 
+[id="ocp-4-8-ip-failover-support"]
+==== IP failover support on {product-title}
+
+IP failover is now supported on {product-title} clusters on bare metal. IP failover uses Keepalived to host a set of externally accessible Virtual IP (VIP) addresses on a set of hosts. Each VIP is only serviced by a single host at a time. Keepalived uses the Virtual Router Redundancy Protocol (VRRP) protocol to determine which host, from the set of hosts, services which VIP. If a host becomes unavailable or if the service that Keepalived is watching does not respond, the VIP is switched to another host from the set, which means a VIP is always serviced as long as a host is available.
```

Comment:
```suggestion
IP failover is now supported on {product-title} clusters on bare metal. IP failover uses Keepalived to host a set of externally accessible Virtual IP (VIP) addresses on a set of hosts. Each VIP is only serviced by a single host at a time. Keepalived uses the Virtual Router Redundancy Protocol (VRRP) protocol to determine which host, from the set of hosts, services which VIP. If a host becomes unavailable, or if the service that Keepalived is watching does not respond, the VIP is switched to another host from the set, which means a VIP is always serviced as long as a host is available.
```
This is a place where I think our style guide might suggest an extra comma, but I am not 100% sure because that guidance is still a bit unintuitive to me :)

Reply:
Could consider finding a way to split "which means a VIP is always serviced as long as a host is available" off into its own sentence for a less complex sentence here?